src/components/project/project.service.ts

```diff
@@ -526,7 +526,15 @@ export class ProjectService {
     const label = `${filter.type ?? ''}Project`;
     const projectSortMap: Partial<Record<typeof input.sort, string>> = {
       name: 'lower(prop.value)',
+      sensitivity: 'sensitivityValue',
     };
+
+    const sensitivityCase = `case prop.value 
+    when 'High' then 3
+    when 'Medium' then 2
+    when 'Low' then 1
+    end as sensitivityValue`;
```

Comment:
Can a `case` expression not be used in an `orderby` statement?

Reply:
not that i've seen, it's usually inside a RETURN or a WITH in the examples.  I played around with it in ORDERBY and couldn't get it to work.